tests/nightly/classic-ubuntu-core-transition-auth/task.yaml

```diff
@@ -20,9 +20,22 @@ debug: |
     . "$TESTSLIB/changes.sh"
     snap change "$(change_id 'Transition ubuntu-core to core')" || true
 
+restore: |
+    # shellcheck source=tests/lib/files.sh
+    . "$TESTSLIB/files.sh"
+
+    clean_file /root/.snap/auth.json
```

Comment:
Mild preference to have executables over functions for the reasons of avoiding the import, avoiding shared state with the test.

Reply:
This is a good suggestion, I'll try to move those functions to executables in a following pr. This change will affect many tests.